xonsh/webconfig/index.html

```diff
@@ -11,7 +11,6 @@
 <body>
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
     <a class="navbar-brand" href="#">
-        <img src="/js/xonsh_sticker.svg" width="30" height="30" class="d-inline-block align-top" alt="">
```

Comment:
I suggest adding back this image while optimizing svg file for its size. svg will look good on all sizes of displays.